modules/dists/StencilDist.chpl

```diff
@@ -351,6 +365,9 @@ class LocStencilArr {
   var locRADLock: atomicbool; // This will only be accessed locally
                               // force the use of processor atomics
 
+  var recvBufs, sendBufs : [locDom.NeighDom] [locDom.bufDom] eltType;
```

Comment:
Add a TODO note to give recvBufs, sendBufs void type when !stencilDistAllowPackedUpdateFluff ?